browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -48,6 +48,11 @@ int OnBeforeURLRequest_StaticRedirectWorkForGURL(
       URLPattern::SCHEME_HTTP | URLPattern::SCHEME_HTTPS, kCRXDownloadPrefix);
   static URLPattern autofill_pattern(
       URLPattern::SCHEME_HTTPS, kAutofillPrefix);
+  static URLPattern gvt1_pattern(
```

Comment:
this also matches the domains that we use for redirecting to crlsets.brave.com. does crlsets still work correctly with this change? it seems like it would work only if the crlsets matching happened before this matching

Reply:
crlsets pattern: https://github.com/brave/brave-browser/pull/8035/files#diff-1561fc2f445953b6e5ac326a56299568R4

Reply:
There is an issue to update `crlsets.brave.com` to `redirector.brave.com` - https://github.com/brave/brave-browser/issues/4634 - The domains point to the same endpoint. I'll pick that up after this PR.

Reply:
ok, if crlsets.brave.com is the same as redirector.brave.com then this seems like it shouldn't break crlsets.